packages/codegen/src/codegen/.hygen/templates/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "commonjs"
```

Comment:
This file is needed so node knows that template files are commonjs instead of ESM